recipe/patches/0016-cross-compile-darwin.patch

```diff
@@ -103,5 +103,5 @@ index d5bbeb90d5..e3354101e6 100644
              else:
                  ret = 1
 -- 
```

Comment:
@isuruf is this considered a change in the end of file line ending? If so, how can I avoid it? https://github.com/conda-forge/python-feedstock/blob/defb66f60454cfdf90c013837b5a1385aff4d8f9/recipe/patches/README.md?plain=1#L12